lib/msf/core/exploit/exe.rb

```diff
@@ -17,6 +17,7 @@ def initialize(info = {})
     register_advanced_options(
       [
         OptBool.new('EXE::EICAR',    [false, 'Generate an EICAR file instead of regular payload exe']),
+		OptBool.new('EXE::Expire',   [false, 'Expire payload after 1 month']),
```

Comment:
Use space for indentation please.

Reply:
Maybe make this a configurable option? ExpireTime ?

Reply:
Can we kick this up to a higher level so it is not limited to exe?